src/GameLogic/PlayerActions/Items/SimpleItemCraftingHandler.cs

```diff
@@ -139,7 +163,9 @@ protected override async ValueTask<List<Item>> CreateOrModifyResultItemsAsync(IL
             {
                 foreach (var item in referencedItem.Items)
                 {
+                    var previousMaxDurability = item.GetMaximumDurabilityOfOnePiece();
                     item.Level += craftingResultItem.AddLevel;
+                    item.Durability = item.GetMaximumDurabilityOfOnePiece() * item.Durability / previousMaxDurability;
```

Comment:
An interesting detail. So that the durability fraction before and after stays the same.
https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/Source/ChaosBox.cpp#L1326